src/WebJobs.Extensions.ApiHub/Table/TableBinding.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Azure.ApiHub.Sdk.Table;
+using Microsoft.Azure.WebJobs.Extensions.ApiHub.Extensions;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.Host.Protocols;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ApiHub.Table
+{
+    internal class TableBinding : IBinding
+    {
+        private ParameterInfo Parameter { get; }
+        private TableConfigContext ConfigContext { get; }
+
+        public TableBinding(
+            ParameterInfo parameter, 
+            TableConfigContext configContext)
+        {
+            ValidateParameter(parameter);
+
+            Parameter = parameter;
+            ConfigContext = configContext;
+        }
+
+        public bool FromAttribute
+        {
+            get { return true; }
+        }
+
+        public Task<IValueProvider> BindAsync(BindingContext context)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException("context");
+            }
+
+            return BindAsync(null, context.ValueContext);
+        }
+
+        public Task<IValueProvider> BindAsync(object value, ValueBindingContext context)
+        {
+            if (context == null)
```

Comment:
You're not using value anywhere here, which means you're not yet supporting string invoke scenarios which are how our portal/Dashboard "Run" button functionality works.